src/components/home/Post.tsx

```diff
@@ -2,8 +2,11 @@
 import React, { useEffect, useRef, useState } from 'react';
 import {jsx, SxStyleProp} from 'theme-ui';
 import {AiOutlineExpandAlt} from 'react-icons/ai';
```

Comment:
Just a style thing, are we doing space or no space for curly brackets

Reply:
We can refactor the styles later but I guess we'll do space?